elastic/kibana
# [Security Solutions] Fix  CellActions component should hide ShowTopN action for nested fields

issue: https://github.com/elastic/kibana/issues/150347

## Summary
CellActions component should hide `ShowTopN` action for nested fields because Lens doesn't support it.

* Add validations to exclude field that are not supported by Lens

**After:**
<img width="1533" alt="Screenshot 2023-06-30 at 14 26 09" src="https://github.com/elastic/kibana/assets/1490444/e3318e05-6a07-44e6-9eb8-861670ecb161">


### How to test
* Go to Security Solutions Alerts page
* Add `Endpoint.policy.applied.artifacts.global.identifiers.name` field
* Check if `ShowTopN` is displayed

### Checklist

Delete any items that are not applicable to this PR.
